libs/ml/include/ml/utilities/graph_builder.hpp

```diff
@@ -261,14 +273,14 @@ void BuildNodeAndInsertTrainables(NodeSaveableParams<T> const &nsp, std::string
     g->AddTrainable(node, name);
     break;
```

Comment:
I wonder if we could split this header file to both cpp and hpp file, and move all Ops includes to cpp file (and implementation of all functions that are declared here). This way we can reduce include amount in this particular header and reduce its impact on unit test .cpp files without merging many UT .cpp files to a large one.

Reply:
this header is also included in vm-modules Model (VMModel) but it actually does not need the whole scope of includes too and could be slightly refactored; the same for vm-modules Graph.